### ISSUE TITLE
👑[Enhancement] 希望能支持Windows arm版

### ISSUE THREAD

**Issue by MDeath** [BeyondDimension/SteamTools#2114]
希望能把Windows arm版提上日程，不然x86x64版有点耗电

**Comment by AigioL** [comment]
我们有一个 win10-arm64 的配置文件，但是没有 arm64 的设备进行测试所以一直没发
https://github.com/BeyondDimension/SteamTools/blob/b37864dd18eab3c83d321cf46a651248e0646cf1/src/ST.Client.Desktop.Avalonia.App/Properties/PublishProfiles/win-arm64.pubxml#L6-L17

**Comment by MDeath** [comment]
没编译过(ˉ﹃ˉ)能帮帮编译一下吗

**Comment by AigioL** [comment]
这里不能上传 7z 的包，且单个文件有 25 MB 的限制，下载后后缀名删掉 .zip 解压两次就行了

ARM64 的版本可能在使用 SAM.API 的部分上有点问题，比如库存游戏的 管理成就/添加到挂时长列表

[win-arm64.zip.001](https://github.com/BeyondDimension/SteamTools/files/9771748/win-arm64.zip.001.zip)
[win-arm64.zip.002](https://github.com/BeyondDimension/SteamTools/files/9771749/win-arm64.zip.002.zip)

**Comment by MDeath** [comment]
> 这里不能上传 7z 的包，且单个文件有 25 MB 的限制，下载后后缀名删掉 .zip 解压两次就行了
> 
> ARM64 的版本可能在使用 SAM.API 的部分上有点问题，比如库存游戏的 管理成就/添加到挂时长列表
> 
> [win-arm64.zip.001](https://github.com/BeyondDimension/SteamTools/files/9771748/win-arm64.zip.001.zip) [win-arm64.zip.002](https://github.com/BeyondDimension/SteamTools/files/9771749/win-arm64.zip.002.zip)

试了试会闪退

**Comment by someash** [comment]
我的win11arm设备在微软商店里显示支持我的设备，但是打开后是纯色的，对应位置点击有反馈（关闭，退出等），但是没有显示，楼上发的那个包闪退

**Comment by EROMIU** [comment]
我的情况和前面的老哥一样，打开之后显示纯色，没法使用，希望大佬能早日适配

**Comment by MDeath** [comment]
> 我的win11arm设备在微软商店里显示支持我的设备，但是打开后是纯色的，对应位置点击有反馈（关闭，退出等），但是没有显示，楼上发的那个包闪退

我用的就是win11商店的，实际是x64转译的和其他win版本是一样的

![image](https://user-images.githubusercontent.com/42289879/196114347-fcd5f039-ca16-444f-99f8-6d0e42bef781.png)

**Comment by someash** [comment]
> > 我的win11arm设备在微软商店里显示支持我的设备，但是打开后是纯色的，对应位置点击有反馈（关闭，退出等），但是没有显示，楼上发的那个包闪退
> 
> 我用的就是win11商店的，实际是x64转译的和其他win版本是一样的
> 

其实win11arm商店下的watt的功能应该是正常的，因为打开的时候还能检测我电脑上正在运行的steam账户，就是显示有问题，纯色，如下图所示，不知道是什么问题
![image](https://user-images.githubusercontent.com/32922655/196172269-0e528935-0404-405d-8dd9-9d75a6fbe327.png)

**Comment by AigioL** [comment]
可能与 https://github.com/AvaloniaUI/Avalonia/issues/8776 https://github.com/AvaloniaUI/Avalonia/issues/9353 相关
我们有一台 Mac Mini M1 的设备最近在虚拟机中调试准备解决这个问题
可能升级到 AvaloniaUI 11 后能正常

**Comment by luojunyuan** [comment]
显示纯色的人可以通过删除目录下的av_libglesv2.dll后应该就能正常打开

**Comment by afunc233** [comment]
> 显示纯色的人可以通过删除目录下的av_libglesv2.dll后应该就能正常打开

是的 可以用的 不过 opengl 就用不了了  看  [#10405](https://github.com/AvaloniaUI/Avalonia/issues/10405)  我感觉应该是 设备对 opengl api 版本支持的问题 ，我这里有个设备，opengl api 指支持 1.1
